flytekit/extras/sqlalchemy/task.py

```diff
@@ -0,0 +1,73 @@
+import os
+import typing
+from dataclasses import dataclass
+
+import pandas as pd
+from sqlalchemy import create_engine
+
+from flytekit import FlyteContext, kwtypes
+from flytekit.core.base_sql_task import SQLTask
+from flytekit.core.python_function_task import PythonInstanceTask
+from flytekit.types.schema import FlyteSchema
+
+
+@dataclass
+class SQLAlchemyConfig(object):
+    """
+    Use this configuration to configure task. String should be standard
+    sqlalchemy connector format
+    (https://docs.sqlalchemy.org/en/14/core/engines.html#database-urls).
+    Database can be found:
+      - within the container
+      - or from a publicly accessible source
+
+    Args:
+        uri: default sqlalchemy connector
+    """
+
+    uri: str
```

Comment:
where do we get the username + password.  I think we should use 
Flyte secrets to access it (can be optional ofcourse)
https://docs.flyte.org/projects/cookbook/en/latest/auto_core_intermediate/use_secrets.html#use-secrets-in-a-task